components/Feed/index.js

```diff
@@ -1,34 +1,63 @@
 import React, { Component } from 'react'
 import { graphql, compose } from 'react-apollo'
+import { nest } from 'd3-collection'
+import { timeFormat } from '../../lib/utils/format'
 import { css } from 'glamor'
 import gql from 'graphql-tag'
 import Loader from '../../components/Loader'
 import Link from '../Link/Href'
+import withT from '../../lib/withT'
+import StickyHeader from './StickyHeader'
+import PropTypes from 'prop-types'
 
 import {
   Center,
   TeaserFeed,
   Interaction,
   mediaQueries
 } from '@project-r/styleguide'
+import Button from '@project-r/styleguide/lib/components/Button'
```

Comment:
I think you can just add `Button` to the styleguide import statement above.

Reply:
👍